gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/service/ServiceDistributionPluginTests.groovy

```diff
@@ -497,8 +497,10 @@ class ServiceDistributionPluginTests extends GradleTestSpec {
         def classpathJar = file('dist/service-name-0.0.1/service/lib/').listFiles()
                 .find({ it.name.endsWith("-manifest-classpath-0.0.1.jar") })
         classpathJar.exists()
-        readFromZip(classpathJar, "META-INF/MANIFEST.MF")
-                .contains('Class-Path: guava-19.0.jar produces-manifest-') // etc
+        String fromZip = readFromZip(classpathJar, "META-INF/MANIFEST.MF")
+        fromZip.contains('Class-Path:')
+        fromZip.contains('guava-19.0.jar')
+        fromZip.contains('produces-manifest-')
```

Comment:
```
 .contains('Class-Path: guava-19.0.jar produces-manifest-')
```

the old version required that `guava-19.0.jar` was the first entry. when using `runtimeClasspath` it changed to say `main main guava-19.0.jar`, which i assume means that the main directories were included in the classpath.

which is technically correct that it's the full classpath, but if you were expecting to only have the JAR files, then we'd need to filter the file collection to remove certain entities.